pages/Real-world usage.md

```diff
@@ -82,3 +89,9 @@ defmodule MyApp.Redix do
   end
 end
 ```
+
+And then to use the new wrapper in your appplication:
+```elixir
+MyApp.Redix.command(~w(PING))
+```
```

Comment:
One very small nitpick: I usually use a blank newline before and after code blocks :)